packages/fingerprint-injector/src/fingerprint-injector.ts

```diff
@@ -1,4 +1,5 @@
 import { readFileSync } from 'fs';
+import CDP from 'chrome-remote-interface';
```

Comment:
This should likely be a `type` import. 

Nonetheless, can you add `chrome-remote-interface` as an optional `peerDependency` (same as Playwright and Puppeteer) to `fingerprint-injector`'s [`package.json`](https://github.com/apify/fingerprint-suite/blob/caf240e4938d9971e73ee5c37e7ad3598af61039/packages/fingerprint-injector/package.json#L23)?